release-notes/20230508-ovn-gw-node.md

```diff
@@ -0,0 +1,2 @@
+<!-- markdownlint-disable MD041 -->
+Fix issues while spawning Gateway nodes during cloud prepare for clusters deployed on OpenStack environment running OVN-Kubernetes CNI.
```

Comment:
I still don't think present tense makes sense, ie "Fixed issues ....". "Fix issues" sounds like it's still pending or an instruction to do so. But that's my opinion...

Reply:
Exactly, I had the same thought. But when I had a quick look at the existing release notes[1], I see that it uses "Fix ..." 

[1] https://submariner.io/community/releases/

Reply:
Some use "Fixed". I think we should change all "Fix" to "Fixed".